changelogs/fragments/52045-keyed-group-features.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- add keyed_groups option to add groups under a common parent
```

Comment:
we could make this a bit clearer for users:

- keyed_groups now has a 'parent' keyword that allows assigning all generated groups to the same parent group

Reply:
I'll use that wording, but with the only modification of 'parent' -> 'parent_group' because this is the key being used

Reply:
works4me, it was more a 'direction suggestion' than the required wording, just trying to make it so its easier for uninformed user to figure out what the new feature is by reading the changelog